better_bookmarks/js/content_script.js

```diff
@@ -0,0 +1,28 @@
+var port = chrome.runtime.connect();
+
+window.addEventListener("message", (event) => {
+    if (event.source != window)
+        return;
+
+    if (event.data.type && (event.data.type == "FROM_PAGE")) {
+        console.log("Content script received: " + event.data.text);
+        port.postMessage(event.data.text);
+    }
+}, false);
+
+const title = document.getElementsByTagName("title")[0];
+const metas = document.getElementsByTagName("meta");
+const url = document.URL;
+for (let i = 0; i < metas.length; i++) {
+    const meta = metas[i];
+    if (meta.name.toLowerCase() == "description"){
+        var description = meta.content;
```

Comment:
We should put a ```var description;``` declaration at the top of the file or at the top of the for loop instead of relying on JavaScript's default hoisting behavior.